server/signup.php

```diff
@@ -1,12 +1,50 @@
 <?php
 
-$values = [];
-foreach ($_REQUEST as $key => $value) {
-  $values += [htmlspecialchars($key) => htmlspecialchars($value)];
+function getLitmusQuestions() {
+  require 'config.php';
+	$conn = $DB_CONN;
 
-  // Add this content to the database
+	$questionStatement = $conn->prepare('SELECT litmusQuestionId, string, orderIndex, tag, required FROM LitmusQuestion
+        WHERE archived = FALSE
+      	ORDER BY orderIndex');
+  $questionStatement->execute();
+  $questions = $questionStatement->fetchAll();
+
+  $optionStatement = $conn->prepare('SELECT possibleAnswerId, string, orderIndex from PossibleAnswer
+        WHERE litmusQuestionId = ?
+        AND archived = FALSE
+        ORDER BY orderIndex');
+
+  foreach ($questions as $question) {
```

Comment:
It might be better to perform these two SQL statements all in one rather than getting the questions and then getting the options for each question in the for loop. It's the difference between 1 database call and 11+. Something like:

SELECT lq.litmusQuestionId, pa.possibleAnswerId, pa.string AS possibleAnswerText, pa.orderIndex AS possibleAnswerOrderIndex,
	lq.string AS litmusQuestionText, lq.orderIndex AS litmusQuestionOrderIndex, lq.required AS required
FROM PossibleAnswer pa
JOIN LitmusQuestion lq ON pa.litmusQuestionId = lq.litmusQuestionId
WHERE lq.archived = FALSE AND pa.archived = FALSE
ORDER BY lq.orderIndex

Then sort it out into a dictionary in the PHP. That way we only do 1 database call.